### PR TITLE
Fix Tezos Delegation direction

### PR DESCRIPTION
**Pull request by kolya182**
Fixes https://github.com/trustwallet/blockatlas/issues/994

### CODE REVIEW

File: platform/tezos/model.go [hunk 1]
```diff
@@ -67,16 +67,18 @@ func (t *Transaction) ErrorMsg() string {
...
+			return blockatlas.AnyActionUndelegation
+		}
 	}
 
-	if t.Delegate != "" && t.Receiver == "" {
-		return blockatlas.AnyActionUndelegation
-	}
-
-	return blockatlas.AnyActionDelegation
+	return ""
```
Comment (vikmeup) [inline platform/tezos/model.go:81]: why empty string?

File: platform/tezos/model_test.go [hunk 2]
```diff
@@ -45,19 +45,22 @@ func TestTransaction_Status(t *testing.T) {
...
+		out     blockatlas.KeyTitle
 	}{
-		{"Delegation", Transaction{Delegate: "", Receiver: addr1}, blockatlas.AnyActionDelegation},
-		{"Undelegation", Transaction{Delegate: addr1, Receiver: ""}, blockatlas.AnyActionUndelegation},
-		{"Delegation", Transaction{Delegate: "", Receiver: ""}, blockatlas.AnyActionDelegation},
-		{"Delegation", Transaction{Delegate: addr1, Receiver: addr1}, blockatlas.AnyActionDelegation},
+		{"Delegation", addr1, Transaction{Sender: addr1, Delegate: addr2, Receiver: "", Type:TxTypeDelegation}, blockatlas.AnyActionDelegation},
+		{"Undelegation", addr1, Transaction{Sender: addr1, Delegate: "", Receiver: addr2, Type:TxTypeDelegation}, blockatlas.AnyActionUndelegation},
+		{"Unknown type", addr1, Transaction{Sender: addr1, Delegate: addr1, Receiver: addr1}, ""},
+		{"Unknown type", addr1, Transaction{Sender: addr1, Delegate: addr2, Receiver: addr1}, ""},
```
Comment (vikmeup) [inline platform/tezos/model_test.go:56]: why there is unknown type at all?

File: platform/tezos/model.go [hunk 3]
```diff
@@ -67,16 +67,18 @@ func (t *Transaction) ErrorMsg() string {
 	}
 }
 
-func (t *Transaction) Title() blockatlas.KeyTitle {
-	if t.Delegate == "" && t.Receiver != "" {
-		return blockatlas.AnyActionDelegation
+func (t *Transaction) Title(address string) blockatlas.KeyTitle {
```
Comment (vikmeup) [inline platform/tezos/model.go:70]: does not seem unintuitive to pass address to Title function

File: platform/tezos/model.go (hunk 1 above)
Comment (kolya182) [reply platform/tezos/model.go]: https://github.com/trustwallet/blockatlas/pull/995/commits/bb0455b3292ea82ea0c624ba4acb8124ced0ea05

File: platform/tezos/model_test.go (hunk 2 above)
Comment (kolya182) [reply platform/tezos/model_test.go]: https://github.com/trustwallet/blockatlas/pull/995/commits/bb0455b3292ea82ea0c624ba4acb8124ced0ea05

File: platform/tezos/model.go (hunk 3 above)
Comment (kolya182) [reply platform/tezos/model.go]: So far only one way i see how we can determine type

**Pull request merged**